CONTRIBUTING.md

```diff
@@ -13,16 +13,23 @@ improvement.
 When you create your PR, please tag _Craig Baumgarten_ or 
 _Serhii Mamontov_ as a reviewer.
 
-> **NOTE** The pull-request process makes things efficient, and allows 
+> **NOTE** The pull request process makes things efficient, and allows 
 the whole team to participate. If a pull request doesn’t work for you,
 just email Craig or Serhii and they will create one for you.
 
 ## Coding Standards
 
-Repository bundled with ESLint configuration (`.eslintrc`) which will
-warn about any coding style inconsistency with standards defined by
+The repository is bundled with an ESLint configuration (`.eslintrc`) which will
+warn you of any coding style inconsistency with standards defined by
 Airbnb's shared configuration.  
-Make sure to resolve all warnings before open merge pull request.
+Make sure to resolve all warnings before opening a pull request.
```

Comment:
Add a blank line before this so it's on its own line (Markdown ignores single carriage returns).

Reply:
Addressed in e151d1a4126d46af1e3e665953d942b3ee4c9155